packages/react/transform/crates/swc_plugin_dynamic_import/lib.rs

```diff
@@ -196,6 +217,7 @@ where
           text: format!("webpackChunkName: \"{}-{}\"", str_lit, self.opts.layer).into(),
         },
       );
+      self.has_dynamic_import = true;
```

Comment:
`has_lazy_bundle` may be better because we also have dynamic import in the else branch

```suggestion
      self.has_lazy_bundle = true;
```